### PR TITLE
change message when hooks already exist with correct contents

### LINKED ISSUE #96: Install/update workflow and error message
Our usual workflow is to run composer install when merging/changing branches and composer.json/lock changed. Using the default config here will run ```vendor/bin/cghooks add --ignore-lock``` each time and print ```No hooks were added. Try updating```

I'd like to have a change there: instead of error when hooks exist, check if they are the same as the hook that would be created, and print a different message like ```All hooks are up to date```, to prevent confusing new people on the project, I would not want them to randomly run composer update.

What do you think?

### PR DESCRIPTION
**Pull request by black-silence**
Fixes #96, although I'm not sure the test is great. 

I had to kill the push hook, even after installing docker and starting the service make would not finish.

### CODE REVIEW

File: tests/AddCommandTest.php [hunk 1]
```diff
@@ -63,6 +63,27 @@ public function it_does_not_add_hooks_that_already_exist()
...
     }
 
+    /**
+     * @test
+     */
+    public function it_detects_existing_correct_hooks()
+    {
+        $originalHooks = self::$hooks;
+        self::$hooks = [
+            'pre-commit' => '#!/bin/sh' . PHP_EOL . PHP_EOL . 'echo before-commit' . PHP_EOL,
```
[the remark this answers is not in the document]
Comment (black-silence) [reply tests/AddCommandTest.php]: And if the original test hook files were ok, the test it_does_not_add_hooks_that_already_exist would have to be changed.

**Comment by black-silence** [comment]
I'll rebase when all changes are ok.

File: tests/AddCommandTest.php [hunk 2]
```diff
@@ -76,10 +76,7 @@ public function it_detects_existing_correct_hooks()
 
         self::createHooks();
         $this->commandTester->execute([], ['verbosity' => OutputInterface::VERBOSITY_VERBOSE]);
-
-        foreach (array_keys(self::$hooks) as $hook) {
-            $this->assertContains("Hook {$hook} already ok", $this->commandTester->getDisplay());
```
Comment (BrainMaestro) [inline tests/AddCommandTest.php]: I think we still need this, but it should say `[{$hook}] is up to date` like the debug log says

File: src/Commands/AddCommand.php [hunk 3]
```diff
@@ -8,6 +8,7 @@
 class AddCommand extends Command
 {
     private $addedHooks = [];
+    private $keptHooks = [];
```
Comment (BrainMaestro) [inline src/Commands/AddCommand.php]: one last minor thing. can this be renamed to `$upToDateHooks` instead

**Comment by black-silence** [comment]
Pushed clean commit and tag.

Btw I found the pre-push hook problem: https://stackoverflow.com/questions/6901171/is-d-not-supported-by-greps-basic-expressions#comment63406188_6901221

**Comment by BrainMaestro** [comment]
ah no support for PCRE regex. `-P`. Anyway, this looks good, thanks!

**Comment by BrainMaestro** [comment]
can you test it by installing the `dev-master` version? if all is good, I will create a new release

**Comment by black-silence** [comment]
> vendor/bin/cghooks add
All hooks are up to date

**Comment by BrainMaestro** [comment]
Sweet. Thanks!

**Pull request merged**